packages/node-opcua-nodeid/test/test_nodeid.js

```diff
@@ -45,13 +45,13 @@ describe("testing NodeIds", function() {
 
     it("should create a OPAQUE nodeID", function() {
         const buffer = Buffer.alloc(4);
-        buffer.writeUInt32BE(0xdeadbeef, 0);
+        buffer.write("deadbeef", "base64");
 
         const nodeId = new NodeId(NodeIdType.BYTESTRING, buffer, 4);
-        nodeId.value.toString("hex").should.equal("deadbeef");
```

Comment:
Please restore : nodeId.value.toString("hex").should.equal("deadbeef");

Reply:
reverted